src/quo2/components/navigation/top_nav/style.cljs

```diff
@@ -0,0 +1,35 @@
+(ns quo2.components.navigation.top-nav.style)
+
+(defn unread-indicator
+  [unread-count max-value]
+  (let [right-offset (cond
+                       (> unread-count max-value)
+                       -14
+                       ;; Greater than 9 means we'll need 2 digits to represent
+                       ;; the text.
+                       (> unread-count 9)
+                       -10
+                       :else -6)]
+    {:position :absolute
+     :top      -6
+     :right    right-offset
+     :z-index  4}))
```

Comment:
Can't we let the container grow as much as it needs without using negative `right` positioning?

Reply:
This is just for the notification dot. The whole component can grow as much as it wants and it will sit right 👍